v2/cli/init_command_test.go

```diff
@@ -0,0 +1,132 @@
+package main
+
+import (
+	"io/ioutil"
+	"path"
+	"strings"
+	"testing"
+
+	"github.com/brigadecore/brigade/v2/internal/file"
+	"github.com/stretchr/testify/require"
+)
+
+func TestNormalizeLanguage(t *testing.T) {
+	testCases := []struct {
+		name     string
+		expected string
+	}{
+		{name: "typescript", expected: "ts"},
+		{name: "ts", expected: "ts"},
+		{name: "javascript", expected: "js"},
+		{name: "js", expected: "js"},
+	}
+
+	for _, testCase := range testCases {
+		t.Run(testCase.name, func(t *testing.T) {
+			result, err := normalizeLanguage(testCase.name)
+			require.NoError(t, err)
+			require.Equal(t, result, testCase.expected)
+		})
+	}
+}
+
+func TestIDMatch(t *testing.T) {
+	testCases := []struct {
+		id         string
```

Comment:
Nit: Mostly for the sake of consistency, use `name` instead of `id` for your test case structs. That's how it's been done throughout the code base and consistency is best when possible.

Reply:
Ah. Nevermind. I see now that this is meant to be the `id` of a project. That's fine.

But, let's also add a `name` field that describes what's being tested.

By example, `id` might be `aa`, but `name` might be `id is too short`.